faq.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Frequently Asked Questions
+slug: faq
+---
+
+## What is $ustainOSS?
+
+A one-day conference and discussion, were we'll talk about how to sustain open source software. It's not a large conference with talks and keynotes, and it's not an unconference; it's a gather of open source contributors, maintainers, and sustainers; industry leaders and people interested in open source; and friends. The goal is to understand the long term viability of open source as a model for producing software.
+
+## How is it written? (Sustain OSS, $ustainoss, $ustainOSS, $ustain OSS)
+
+SustainOSS is fine. But anything really works.
```

Comment:
The itself is "$ustain". The domain and twitter is `sustainoss` so it is more descriptive plus it is impossible to acquire the domain and twitter account.